major_event_log/models.py

```diff
@@ -30,6 +30,12 @@ class Event(models.Model):
                                     "Name will appear as the Reporting Agent")
     contact_email = models.EmailField()
 
+    def is_success(self):
+        if self.get_outcome_display() == "Success":
```

Comment:
It would be wise for us to rework the `OUTCOME_CHOICES` for this

Currently we have 
```python
OUTCOME_CHOICES = (
    ("http://purl.org/NET/UNTL/vocabularies/eventOutcomes/#success",
     "Success"),
    ("http://purl.org/NET/UNTL/vocabularies/eventOutcomes/#failure",
     "Failure"),
)
```

Which means that `"Success"` is not contained in a single location. Furthermore, I am not sure how I feel about comparing the labels. Comparing the value of `outcome` seems to be more appropriate.

Transforming the code above into this will allow us to do this easily.
```python
SUCCESS = "http://purl.org/NET/UNTL/vocabularies/eventOutcomes/#success"
FAILURE = "http://purl.org/NET/UNTL/vocabularies/eventOutcomes/#failure"
OUTCOME_CHOICES = (
    (SUCESS,  "Success"),
    (FAILURE, "Failure"),
)
```

Then you can compare `SUCCESS` to the value of `outcome` to determine if it is successful or not.

Does that make sense?